scripts/quarantine.yaml

```diff
@@ -139,3 +139,21 @@
   platforms:
     - nrf54h20dk@0.9.0/nrf54h20/cpuapp
   comment: "https://nordicsemi.atlassian.net/browse/NCSDK-35605"
+
```

Comment:
@kapi-no  FYI

Reply:
Hi @nomo-nordic, @magp-nordic,
How long is `nrf52kbd/nrf52832` expected to stay in the quarantine?

Reply:
@MarekPieta it should be removed in  https://github.com/nrfconnect/sdk-nrf/pull/24631